### PR TITLE
Power Platform Demo Tenant Setup

### PR DESCRIPTION
**Pull request by Grant-Archibald-MS**
Provide the ability to:
1. Create demo users
2. Create security group for license assignment to makers
3. Assign users to Maker security group
4. Create development environment for makers

### CODE REVIEW

File: quickstarts/103-demo-tenant/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Demo Tenant Example (103 level)
```
Comment (mawasile) [inline quickstarts/103-demo-tenant/README.md:1]: We generate docs. You need to crete .tmpl readme file like in other quickstarts

**Review by mawasile — CHANGES_REQUESTED** [review]

File: quickstarts/103-demo-tenant/README.md (hunk 1 above)
Comment (Grant-Archibald-MS) [reply quickstarts/103-demo-tenant/README.md]: Added [README.md.tmpl](https://github.com/Grant-Archibald-MS/power-platform-terraform-quickstarts/blob/grant-archibald-ms/demo-tenant/quickstarts/103-demo-tenant/README.md.tmpl) and removed README.md

Comment (mawasile) [reply quickstarts/103-demo-tenant/README.md]: Sorry, forgot to tell you that you still have to run docs generation manually (we don't do that automatically in the repo at the moment)

https://github.com/Grant-Archibald-MS/power-platform-terraform-quickstarts/tree/grant-archibald-ms/demo-tenant/tools/quickstartgen 

Once everything is ok, you should see your quick start linked at the bottom of the main readme.md file

Comment (Grant-Archibald-MS) [reply quickstarts/103-demo-tenant/README.md:2]: @mawasile `go run quickstartgen.go` run and README.md generated

**Review by mawasile — APPROVED** [review]

**Pull request merged**